explorer/nodewatch/nodewatch/RoutesFacade.py

```diff
@@ -94,17 +94,14 @@ def custom_filter(descriptor):
 		if order == 'random':
 			random.shuffle(nodes)
 
-		if limit is not None:
-			nodes = nodes[:limit]
-
-		return nodes
+		return nodes if limit == 0 else nodes[:limit]
```

Comment:
if not limit. ?

Reply:
https://github.com/symbol/product/blob/9a18900865ffe7c692eb0e796baf73da89446d46/explorer/nodewatch/nodewatch/__init__.py#LL107C3-L107C3

If that is not the limit, it will set it to 0.